CT-web/src/main/kotlin/com/chuseok22/ctweb/infrastructure/config/ComponentScanConfig.kt

```
@@ -0,0 +1,11 @@
+package com.chuseok22.ctweb.infrastructure.config
+
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ComponentScan(basePackages = [
+  "com.chuseok22.*"
+])
+class ComponentScanConfig {
+}
```

detekt (1.23.8)

[warning] The class or object ComponentScanConfig is empty.

(detekt.empty-blocks.EmptyClassBlock)